### PR TITLE
[IA-2584] Scala steward updates

### PR DESCRIPTION
**Pull request by rtitle**
I pulled all the `scala-steward` PRs into 1 PR. Haven't tested yet, just wanted to see if it all works.

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -56,7 +56,7 @@ object Dependencies {
 
   val logbackClassic: ModuleID =  "ch.qos.logback"              % "logback-classic" % "1.2.3"
   val scalaLogging: ModuleID =    "com.typesafe.scala-logging"  %% "scala-logging"  % scalaLoggingV
-  val swaggerUi: ModuleID =       "org.webjars"                 % "swagger-ui"      % "3.40.0"
+  val swaggerUi: ModuleID =       "org.webjars"                 % "swagger-ui"      % "3.44.0"
```
Comment (rtitle) [inline project/Dependencies.scala:59]: Does updating swagger require a code change?

**Comment by Qi77Qi** [comment]
great thanks! wonder how you did that :)

**Review by Qi77Qi — APPROVED** [review]
:shipit:

**Comment by rtitle** [comment]
> wonder how you did that

Manual `git merge` but I thought about making a script. :)

Will do a little manual testing, then ticket+merge this if it looks good.

**Comment by Qi77Qi** [comment]
liquibase-core upgrade might be tricky, there was some issue when PR was merged to dev last time.

swagger-ui upgrade might needs some attention too

**Comment by rtitle** [comment]
@Qi77Qi automation tests passed.. I also deployed to my fiab and created a runtime + galaxy successfully. Swagger looks good too.

Do you remember what the `liquibase` issue was? I can't find it in slack history..

**Comment by Qi77Qi** [comment]
@rtitle oh nvm, I thought it's liquibase...but it's mysql https://github.com/DataBiosphere/leonardo/pull/1842

**Pull request merged**